src/main/php/PHPMD/Rule/CleanCode/StaticAccess.php

```diff
@@ -66,7 +66,24 @@ public function apply(AbstractNode $node)
 
     protected function isExcludedFromAnalysis($className, $exceptions)
     {
-        return in_array(trim($className, " \t\n\r\0\x0B\\"), $exceptions);
+        $className = trim($className, " \t\n\r\0\x0B\\");
+
+        if (in_array($className, $exceptions)) {
+            return true;
+        }
+
+        $wildcardExceptions = array_filter($exceptions, function ($exception) {
+            return strpos($exception, '*') !== false;
+        });
+        foreach ($wildcardExceptions as $wildcardException) {
+            $wildcardException = str_replace('*', '.*', $wildcardException);
+            $wildcardException = str_replace('\\', '\\\\', $wildcardException);
```

Comment:
This will be easier to clean up once we merge it in to 3.x.

```suggestion
            $wildcardException = str_replace(array('*', '\\'), array('.*', '\\\\'), $wildcardException);
```